examples/plotting/file/sparkline.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from bokeh.charts import TimeSeries
+from bokeh.models import (
+    Legend,
+    Circle,
+    HoverTool,
+    ColumnDataSource,
+    GlyphRenderer
```

Comment:
Use a trailing comma here in similar contexts. This will make future changes easier on the diff, e.g. adding new import will require just one addition line, not removal and two additions.